src/rebar_prv_erlydtl_compiler.erl

```diff
@@ -125,27 +123,40 @@ init(State) ->
                                                                {opts, []}])),
     {ok, State1}.
 
-do(Config) ->
+do(State) ->
     ?INFO("Running erlydtl...", []),
-    MultiDtlOpts = erlydtl_opts(Config),
+    DtlOpts = proplists:unfold(rebar_state:get(State, erlydtl_opts, [])),
+    App1 = case rebar_state:project_apps(State) of
```

Comment:
So that one is about finding which app to put the compiled templates in? Might like a comment to that effect.